common/fixation_entry.go

```diff
@@ -133,8 +138,8 @@ func (fs *FixationStore) AppendEntry(ctx sdk.Context, index string, block uint64
 		// make sure the new entry's block is not smaller than the latest entry's block
 		if block < latestEntry.GetBlock() {
 			details := map[string]string{
-				"latestBlock": strconv.FormatUint(latestEntry.GetBlock(), 10),
-				"block":       strconv.FormatUint(block, 10),
+				"latestBlock": strconv.Itoa(int(latestEntry.GetBlock())),
```

Comment:
why is it better? we are casting uint to int just to use ITOA, is the convert more safe?

Reply:
just less verbose.
I can revert it.